Tryouts/Messaging/Client/PublishOptions.cs

```diff
@@ -10,10 +10,11 @@
 // or implied. See the License for the specific language governing permissions
 // and limitations under the License.
 
+using MorganStanley.ComposeUI.Messaging.Core;
+
 namespace MorganStanley.ComposeUI.Messaging.Client;
 
-// TODO: MessagingScope
-public class MessagingScope
+public struct PublishOptions
 {
-    
+    public MessagingScope Scope { get; set; }
```

Comment:
Mutable structs aren't a good idea

Reply:
I have no problem changing it to init only, but since the options will be extended over time, the alternative is a positional record with many optional parameters. It can also be a class, if we don't mind the extra allocation per message just for a parameter object.

Reply:
init only sounds like a much safer choice to me.